packages/relay-sdk/src/index.ts

```diff
@@ -0,0 +1,132 @@
+
+import { RELAY_URL_HABANERO, RELAY_URL_MANZANO } from '@lit-protocol/constants';
+import { assertNotNull, assertStatus, assertString, getProp } from './util';
+
+type SupportedNetworks = 'habanero' | 'manzano';
+
+function getRelayURLByNetwork(network: SupportedNetworks): string {
+    switch (network) {
+        case 'habanero':
+            return RELAY_URL_HABANERO;
+        case 'manzano':
+            return RELAY_URL_MANZANO;
+    }
+}
+
+export class LitRelayClient {
+
+    private readonly baseUrl: string;
+    private readonly apiKey: string;
+
+    private payerSecret: string | undefined = undefined;
+
+    /**
+     * Create a new LitRelayClient instance. Requires that the payer is already registered.
+     * and the the payer secret is known.
+     * 
+     * ```typescript
+     * const client = new LitRelayClient('https://habanero.lit.dev', 'my-api-key', 'my-payer-secret');
+     * ```
+     * 
+     * @param baseUrl
+     * @param apiKey
+     * @param payerSecret
+     */
+    private constructor(baseUrl: string, apiKey: string, payerSecret: string) {
+        this.baseUrl = baseUrl;
+        this.apiKey = apiKey;
+
+        if (payerSecret) {
+            this.payerSecret = payerSecret;
+        }
+    }
+
+    /**
+     * Adds a new payee to the payer's delegation list.
+     * 
+     * ```typescript
+     * const client = await LitRelayClient.connect('habanero', 'my-api-key', 'my-payer-secret');
+     * const result = await client.addPayee('payee-wallet-address');
+     * ```
+     * @param payeeAddress
+     * 
+     * @returns Promise<{ tokenId: string } | Error>
+     */
+    public addPayee(payeeAddress: string): Promise<{ tokenId: string }> {
```

Comment:
Can we change this function to be `addPayees` given that the backend API supports that? Otherwise we will inevitably end up with 2 methods, and this method will end up either calling the `addPayees` method or duplicating the code inside of that method.